lang/en/zoom.php

```diff
@@ -42,6 +42,10 @@
 $string['allmeetings_enable'] = 'Enable all meetings link';
 $string['apikey'] = 'Zoom API key';
 $string['apikey_desc'] = '';
+$string['apiendpoint'] = 'Zoom API Endpoint';
+$string['apiendpoint_desc'] = 'With this setting, you control to which Zoom API endpoint the Zoom activity will connect to. The global API endpoint is working in all setups. The EU API endpoint is intended for Zoom customers only who have a Zoom license with the EU provisioning option. If you have any doubts, please leave the setting set to the global API endpoint.';
```

Comment:
There were a few words that I wanted to adjust. Please check my proposed wording in case I have used words or phrases that do not easily/directly translate.
```suggestion
$string['apiendpoint_desc'] = 'Choose which Zoom API endpoint the Zoom activity will use to connect. The global API endpoint should work for all users. The European Union (EU) API endpoint is only intended for users who have a Zoom license with the EU provisioning option. If you are unsure, use the global API endpoint.';
```

Reply:
As I am not a native speaker, I always appreciate proof-readings.
I will be happy to adopt your changes.